05-JWT-Basics/starter/routes/main.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const router = express.Router();
+
+const { login, dashboard } = require('../controllers/main')
+
+const authMiddleware = require('../middleware/auth');
+const authentificationMiddleware = require('../middleware/auth');
```

Comment:
```suggestion
```
This line is duplicated, you can remove this.

Reply:
Hello! Thank you! Should I create new branch e.g. week8-1 to fix these mistakes or can I do changes to the same week8 assignment and git commit and push it again?

Reply:
@aschey  Hello! Thank you! Should I create new branch e.g. week8-1 to fix these mistakes or can I do changes to the same week8 assignment and git commit and push it again?

Reply:
You can make the changes directly in this branch and then feel free to merge it afterwards.

Reply:
@aschey oh, okay, got it! thank you!